react/features/conference/components/native/Labels.js

```diff
@@ -354,8 +357,9 @@ class Labels extends AbstractLabels<Props, State> {
 function _mapStateToProps(state) {
     return {
         ..._abstractMapStateToProps(state),
+        _aspectRatio: state['features/base/responsive-ui'].aspectRatio,
```

Comment:
If you calculate ```wide``` here instead of passing the ```_aspectRatio``` and let the render method calculate it, you'll get less updates

Reply:
Maybe I'm missing something here, but why would using a boolean which checks a specific value for the prop be better than the prop value itself, since it's immutable? (it's a Symbol).

Reply:
Right, sorry, I missed that detail!